lib/mindwendel/brainstormings.ex

```diff
@@ -67,7 +67,8 @@ defmodule Mindwendel.Brainstormings do
   """
   # See https://stackoverflow.com/questions/53802091/elixir-uuid-how-to-handle-500-error-when-uuid-doesnt-match
   def get_brainstorming!(id) do
-    Repo.get!(Brainstorming, id)
+    id
+    |> get_bare_brainstorming!()
     |> Repo.preload([
       :users,
```

Comment:
all these `preloads` absolutely weren't needed in our case. It's always tough to split up, I created a new function as that seemed least disruptive but yeah there are many options from specifying preloads as an argument and others.

That's one of the reasons why I like to have the `preload` as close to where it is used as possible, as otherwise you're prone to over-fetching. It isn't always easy though and would sometimes infringe on SRP.